AdditionalCosts.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using PriceCalculatorSolution;
+
+namespace CostSolution
+{
+    public class AdditionalCosts
+    {
+        public static int TaxPercentage;
+        public static decimal PackagingAmount;
+        public static decimal TransportAmount;
+        public AdditionalCosts(int _tax, decimal _packagingAmount, decimal _transportAmount)
```

Comment:
there is no a predefined set of expenses, you can define whatever you want, your solution does not support that